packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/TrendingCommunitiesPreview.tsx

```diff
@@ -10,6 +10,14 @@ export const TrendingCommunitiesPreview = () => {
 
   const sortedCommunities = app.config.chains
     .getAll()
+    .filter((community) => {
+      const name = community.name.toLowerCase();
+      //this filter is meant to not include any de facto communities that are actually xss attempts.
+      //It's a way of keeping the front facing parts of the app clean looking for users
+      return (
+        !['"', '>', '<'].includes(name[0]) && !['"', '>', '<'].includes(name[1])
```

Comment:
Unsure, but should we also check for `/` and `'`?

Reply:
@mzparacha Couldn't hurt. Double checked to see if any existing communities had either of those as the first or second characters and there were none. I added a check for the backtick to be filtered as well.